### ISSUE TITLE
Switching to Minecraft PC Loot tables

### ISSUE THREAD

**Issue by Thunder33345** [Ad5001/BetterGen#10]
Basically title; having a similar or same system as mojang current loot table works

The conditions section could get ignored but some basics would be nice

wiki: http://minecraft.gamepedia.com/Loot_table#Tags

low priority but good to take note of and sorted when other higher things get done

**Comment by inxomnyaa** [comment]
https://github.com/ClearSkyTeam/PocketMine-MP/commit/6ef9e616f9b14a8eff1155e567725d85af9ebadf

Loot table generation/randomisation proven to work here.

This could be imported to this plugin.

**Comment by Thunder33345** [comment]
Table gen: https://github.com/ClearSkyTeam/PocketMine-MP/blob/6ef9e616f9b14a8eff1155e567725d85af9ebadf/src/pocketmine/utils/TempLootGenerator.php
loot tables:
https://github.com/ClearSkyTeam/PocketMine-MP/blob/6ef9e616f9b14a8eff1155e567725d85af9ebadf/src/pocketmine/resources/loot_tables/gameplay/fishing.json
https://github.com/ClearSkyTeam/PocketMine-MP/blob/6ef9e616f9b14a8eff1155e567725d85af9ebadf/src/pocketmine/resources/loot_tables/gameplay/fishing/fish.json
https://github.com/ClearSkyTeam/PocketMine-MP/blob/6ef9e616f9b14a8eff1155e567725d85af9ebadf/src/pocketmine/resources/loot_tables/gameplay/fishing/junk.json

**Comment by inxomnyaa** [comment]
Those loot tables are actually taken from the MCPE default addon.

**Comment by Ad5001** [comment]
Btw am I allowed to take this code and adapt it or not?

**Comment by inxomnyaa** [comment]
> This could be imported to this plugin

**Comment by Thunder33345** [comment]
>Btw am I allowed to take this code and adapt it or not?

i dont see anything stopping you like it's already open sourced
but i advise you not to copy it directly due to how it's meant only for fishing with bunch of hotfixes
love the usage of projects FINALLY... Someone used it

**Comment by Ad5001** [comment]
They are pretty useful by using issues ;)

**Comment by inxomnyaa** [comment]
cough assign me

**Comment by Ad5001** [comment]
@thebigsmileXD Well, I can't until you accept the colaboration.

**Comment by inxomnyaa** [comment]
Oh xd i forgot that this is needed

**Comment by Ad5001** [comment]
@thebigsmileXD Btw when you start working on an issue, Move it in the projects tab ;)

**Comment by inxomnyaa** [comment]
kk

**Comment by inxomnyaa** [comment]
![image](https://cloud.githubusercontent.com/assets/8733998/26032426/dc55a64a-3893-11e7-86f4-734d329aecbf.png)

**Comment by inxomnyaa** [comment]
![image](https://cloud.githubusercontent.com/assets/8733998/26032674/2ff57096-3899-11e7-9e75-c7e1bea70c36.png)
See the loottables branch

**Comment by Thunder33345** [comment]
try to close only after it's merged pull #22 is not merged

**Comment by inxomnyaa** [comment]
the problem is the legal part
eula and shit

**Comment by inxomnyaa** [comment]
![image](https://cloud.githubusercontent.com/assets/8733998/26101502/a4f42d4e-3a31-11e7-8331-3a9bb3985190.png)
This might take some fucking time

**Comment by inxomnyaa** [comment]
Nope, fuck this. I will not go through every fucking file and remake all changes

@Ad5001 DIY.
